tests/test_plugin.py

```diff
@@ -31,3 +38,18 @@ def test_stop_plugin_configuration_on_conn_error(mocked_get):
     expect(mocked_config._reportportal_configured is False,
            'The value of the _reportportal_configured is not False.')
     assert_expectations()
+
+
+@patch('pytest_reportportal.RPLogger.handle')
+def test_logger_handle_attachment(mock_handler, logger):
+
+    logger.info("Some info message", attachment="Some info attachment")
+    expect(mock_handler.call_count == 1)
+    expect(getattr(mock_handler.call_args[0][0], "attachment") == "Some info attachment")
+    mock_handler.reset_mock()
+
+    logger.info("Some debug message", attachment="Some debug attachment")
```

Comment:
I think that the second block tests the same functionality and it can be omitted. I would test here a situation when log message comes to the handler without an attachment. 

P.S. You can utilize Pytest's parametrize fixture when your tests are data-driven to avoid code duplication: https://docs.pytest.org/en/latest/parametrize.html

Reply:
Fixed